app/helpers/users_helper.rb

```diff
@@ -0,0 +1,7 @@
+module UsersHelper
```

Comment:
module thì khác gì so với class nhỉ?

Reply:
- Class là bản thiết kế của object, thể hiện đặc điểm chung của một nhóm các object bằng cách tìm ra những thuộc tính và phuơng thức chung của object.
- Module là một tập các lớp, phương thức, hằng số do đó module cũng giống như các class. Chỉ khác là module không thể tạo các đối tượng và không thể kế thừa. Module thường dùng để mix-in(nhúng) vào một hoặc nhiều class, module khác.